sdk/storage/azure-storage-file-datalake/out/azure/storage/filedatalake/_file_system_client.pyi

```diff
@@ -0,0 +1,33 @@
+# Stubs for azure.storage.filedatalake._file_system_client (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from ._data_lake_directory_client import DataLakeDirectoryClient
+from ._data_lake_file_client import DataLakeFileClient
+from ._data_lake_lease import DataLakeLeaseClient
+from ._generated import DataLakeStorageClient
+from ._models import FileSystemProperties, LocationMode, PathPropertiesPaged
+from ._serialize import convert_dfs_url_to_blob_url
+from ._shared.base_client import StorageAccountHostsMixin, parse_connection_str, parse_query
+from azure.core.paging import ItemPaged
+from typing import Any, Optional
+
+class FileSystemClient(StorageAccountHostsMixin):
```

Comment:
It seems like this is the only client that doesn't start with "DataLake" - is that intentional?

Reply:
Yes it's intentional, because ContainerClient is not called BlobContainerClient, and ShareClient is not called FileShareClient.